src/compaction_execute_bench.rs

```diff
@@ -55,10 +55,10 @@ fn open_object_store(options: &Options) -> Result<Arc<dyn ObjectStore>, SlateDBE
 }
 
 #[allow(clippy::panic)]
-pub fn run_compaction_execute_bench() -> Result<(), SlateDBError> {
+pub fn run_compaction_execute_bench(c: Option<CompressionCodec>) -> Result<(), SlateDBError> {
```

Comment:
WDYT about adding a `compression_codec: String` to `Options`, and using that here.